job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy

```diff
@@ -59,25 +71,13 @@ pipeline{
                 ]
             }
         }
-        
-        stage('trigger nightly job 7.59.x') {
-            steps {
-                build job: 'nightly/7.59.x', propagate: false, wait: true, parameters: [
-                        [\$class: 'StringParameterValue', name: 'KIE_GROUP_DEPLOYMENT_REPO_URL', value: 'https://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8443/nexus/service/local/repositories/scratch-release-rhba-7.12/content-compressed'],
-                        [\$class: 'StringParameterValue', name: 'UMB_VERSION', value: '712'],
-                        [\$class: 'StringParameterValue', name: 'PRODUCT_VERSION', value: '7.12.0'],
-                        [\$class: 'StringParameterValue', name: 'DEFAULT_CONFIG_BRANCH', value: '7.59.x'],
-                        [\$class: 'BooleanParameterValue', name: 'SKIP_TESTS', value: true]
-                ]
-            }
-        }
 
         // Kogito prod nightlies
         stage('trigger kogito nightly job 1.11.x') {
             steps {
                 build job: 'kogito.nightly/1.11.x', propagate: false, wait: true, parameters: [
                         [\$class: 'StringParameterValue', name: 'RHBA_MAVEN_REPO_URL', value: 'http://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8081/nexus/content/repositories/rhba-7.12-nightly-with-upstream'],
-                        [\$class: 'StringParameterValue', name: 'RHBA_VERSION_PREFIX', value: "${RHBA_VERSION_PREFIX}"],
+                        [\$class: 'StringParameterValue', name: 'RHBA_VERSION_PREFIX', value: "7.59.0.redhat-"],
```

Comment:
this should be matching the version from 7.59.x branch of https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/7.59.x/pom.xml#L25 -> 7.59.1-SNAPSHOT translate intos 7.59.1.Final, so it is not good to use hardcoded string. Better to keep variable and fix the job parameter or take/create it from `version.kie` property.